website/docs/modules/gather_batch_evidence.md

```diff
@@ -5,25 +5,199 @@ sidebar_position: 4
+  gse: GatherSampleEvidence
+  eqc: EvidenceQC
+  gcnv: TrainGCNV
+  gbe: GatherBatchEvidence
+  cbe: ClusterBatch
+  gse --> gbe
+  eqc --> gbe
+  gcnv --> gbe
+  gbe --> cbe
+  
+  class gbe thisModule
+  class gse, eqc, gcnv inModules
+  class cbe outModules
+```
+
+## Inputs
+This workflow takes as input the read counts, BAF, PE, SD, SR, and per-caller VCF files 
+produced in the GatherSampleEvidence workflow, and contig ploidy and gCNV models from 
+the TrainGCNV workflow.
+The following is the list of the inputs the GatherBatchEvidence workflow takes.
+
+
+#### `batch`
+An identifier for the batch.
+
+
+#### `samples`
+Sets the list of sample IDs. 
+
+
+#### `counts`
+Set to the [`GatherSampleEvidence.coverage_counts`](./gse#coverage-counts) output.
+
+
+#### Raw calls
+
+The following inputs set the per-caller raw SV calls, and should be set 
+if the caller was run in the [`GatherSampleEvidence`](./gse) workflow.
+You may set each of the following inputs to the linked output from 
+the GatherSampleEvidence workflow.
+
+
+- `manta_vcfs`: [`GatherSampleEvidence.manta_vcf`](./gse#manta-vcf);
+- `melt_vcfs`: [`GatherSampleEvidence.melt_vcf`](./gse#melt-vcf);
+- `scramble_vcfs`: [`GatherSampleEvidence.scramble_vcf`](./gse#scramble-vcf);
+- `wham_vcfs`: [`GatherSampleEvidence.wham_vcf`](./gse#wham-vcf).
+
+#### `PE_files`
+Set to the [`GatherSampleEvidence.pesr_disc`](./gse#pesr-disc) output.
+
+#### `SR_files`
+Set to the [`GatherSampleEvidence.pesr_split`](./gse#pesr-split)
+
+
+#### `SD_files`
+Set to the [`GatherSampleEvidence.pesr_sd`](./gse#pesr-sd)
+
+
+#### `matrix_qc_distance`
+You may set it to `1000000`.
+
+
+#### `min_svsize`
+Sets the minimum size of SVs to include. 
+You may set it to `50`. 
+
+
+#### `ped_file`
+A pedigree file describing the familial relationshipts between the samples in the cohort.
+The file needs to be in the 
+[PED format](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format).
+Updated with [EvidenceQC](./eqc) sex assignments, including 
+`sex = 0` for sex aneuploidies. Calls will not be made on sex chromosomes 
+when `sex = 0` in order to avoid generating many confusing calls 
+or upsetting normalized copy numbers for the batch.
+
+
+#### `run_matrix_qc`
+Enables or disables running optional QC tasks. 
+
+
+#### cn.MOPS files
+The workflow needs the following cn.MOPS files.
+
+- `cnmops_chrom_file` and `cnmops_allo_file`: FASTA index files (`.fai`) for respectively non-sex chromosome (autosome) and chromosomes X and Y (allosomes). 
+  The content of the files may read as the following, 
+  and the format is explained [on this page](https://www.htslib.org/doc/faidx.html).
+
+  ```bash
+  chrX    156040895       2903754205      100     101
+  chrY    57227415        3061355656      100     101
+  ```
+
+  You may use the following files for these fields:
+
+  ```json
+  "cnmops_chrom_file": "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/autosome.fai"
```

Comment:
What is a good reference for these?

Reply:
I think it makes most sense to direct users to the JSONs in `inputs/` in general rather than linking to a specific JSON for each input (cluttered, requires more maintenance)

Reply:
I updated it to link to a specific file in the resources JSON; that is not much better than this, but we have ongoing internal discussions on how best to address such inputs. We should not point to the `inputs/` directory without direct references as it will be a needle in a haystack given all the resources and a lot of variable name mismatches.